docs/API.md

```diff
@@ -4,8 +4,13 @@ TODO: make this into a nicer doc
 
 By default `127.0.0.1:1234` - daemon stores the api endpoint multiaddr in `~/.lotus/api`
 
-* `http://[api:port]/rpc/v0` - jsonrpc http endpoint
-* `ws://[api:port]/rpc/v0` - jsonrpc websocket endpoint
+* `http://[api:port]/rpc/v0` - JsonRPC http endpoint
+* `ws://[api:port]/rpc/v0` - JsonRPC websocket endpoint
+* `PUT http://[api:port]rest/v0/import` - import file to the node repo
+  * Requires write permission
```

Comment:
```suggestion
  * Requires admin permission
```

Reply:
No, that one only requires `write` - https://github.com/filecoin-project/lotus/pull/464/files#diff-3049d9a11322263539c11d573869b5baR65

(and I've changed the other one as you could technically import the admin token from the repo with that method..)

Reply:
Ah, got it